src/System.CommandLine/Command.cs

```diff
@@ -10,12 +10,29 @@
 
 namespace System.CommandLine
 {
-    public class Command : Symbol, ICommand, IEnumerable<Symbol>
+    public class Command : 
+        Symbol, 
+        ICommand, 
+        IEnumerable<Symbol>
     {
         private readonly SymbolSet _globalOptions = new SymbolSet();
 
-        public Command(string name, string? description = null) : base(new[] { name }, description)
+        public Command(string name, string? description = null) : base(description)
         {
+            Name = name;
```

Comment:
Won't this add the alias in the Name property setter? Doesn't it make the below AddAlias redundant?